Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -36,6 +38,7 @@ public function store($data)
         $applicant->graduation_year = $data['graduation_year'] ?? null;
         $applicant->university = $data['university_name'] ?? null;
         $applicant->centre_id = $data['centre'];
+        Mail::to($data['email_id'], $data['first_name'])->send(new CodetrekMailApplicant());
```

Comment:
Can you use the mail queue here?
Something like this.

```suggestion
        Mail::queue(new CodetrekMailApplicant($data));
```